src/test/java/com/amashchenko/maven/plugin/gitflow/GitFlowFeatureStartMojoTest.java

```diff
@@ -0,0 +1,77 @@
+package com.amashchenko.maven.plugin.gitflow;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+/**
+ * Created by krs on 2/18/17.
+ */
+public class GitFlowFeatureStartMojoTest {
```

Comment:
The test is in `GitFlowFeatureStartMojo` and cannot be tested without an instance of that class in the same package scope so it only seemed natural - more tests could be added in the future.